float32 x_pos_gnd
float32 y_pos_gnd